lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc

```diff
@@ -442,9 +458,10 @@ function dosomething_signup_user_signup_data_form($form, &$form_state, $signup)
     '#suffix' => '</div></div>',
   );
 
+  $submit_label = ($config['submit_text']) ? $config['submit_text'] : 'Submit';
   $form['actions']['submit'] = array(
     '#type' => 'submit',
-    '#value' => t('Submit'),
+    '#value' => $submit_label,
```

Comment:
well if we want the custom text to be translateable I think maybe `$submit_label` but I think I remember `t()` having issues with translating a variable, or just there's some caveat we need to be aware of...